src/app/views/Layout.styles.ts

```diff
@@ -0,0 +1,94 @@
+import { makeStyles, tokens } from '@fluentui/react-components';
```

Comment:
Given the new designs and the effort needed to make the component look as the design as possible, what is the effort you figure is needed to achieve the migration? Are there lots of CSS tinkering? Does this file also mean there will be a change in the way styling is done (theming)?